upgrade_tests/test_existance_relations/cli/test_location.py

```diff
@@ -0,0 +1,74 @@
+"""Upgrade TestSuite for validating Satellite locations existence and
+associations post upgrade
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: CLI
+
+:TestType: nonfunctional
+
+:CaseImportance: High
+
+:SubType1: installability
+
+:Upstream: No
+"""
+import pytest
+from upgrade_tests.helpers.common import existence
+from upgrade_tests.helpers.existence import compare_postupgrade, pytest_ids
+
+# Required Data
+component = "location"
+loc_id = compare_postupgrade(component, 'id')
+loc_name = compare_postupgrade(component, 'name')
+loc_desc = compare_postupgrade(component, 'description')
+loc_title = compare_postupgrade(component, 'title')
+
+
+# Tests
+@pytest.mark.parametrize("pre,post", loc_id, ids=pytest_ids(loc_id))
+def test_positive_locations_by_id(pre, post):
+    """Test all locations are existing after upgrade by id's
+
+    :id: upgrade-d16b33af-46b3-4413-b018-e7a4926c8bcc
```

Comment:
make sure these are all new, ignore if it is already

Reply:
yes, i have added new test id's for all tests.